README: highlight quick-start and update the contact-us chapter (#17447)
pingcap/tidb: README.md

For more details and latest updates, see [official TiDB blog](https://www.pingcap.com/blog/).

## Quick start

> ### To start using TiDB

Reviewer: Use TiDB

Author: It was referred from k8s https://github.com/kubernetes/kubernetes#to-start-using-kubernetes

Read the [Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/ ), which includes deployment methods using TiUP, Ansible, Docker, and Kubernetes.

> ### To start developing TiDB

Reviewer: Contribute to TiDB

Author: Ditto, referred from k8s https://github.com/kubernetes/kubernetes#to-start-developing-kubernetes

> The [community repository](https://github.com/pingcap/community) hosts all information about building TiDB from sourse, how to contribute code and documentation, how to join special interest group, who to contact about what, etc.

Reviewer 2: ```suggestion
The [community repository](https://github.com/pingcap/community) hosts all information about the TiDB community, including how to contribute to TiDB, how TiDB community is governed,  how special interest groups are organized, etc.
```
